src/tests/ftest/posix/simul.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import distro
+from avocado.core.exceptions import TestFail
+from dfuse_test_base import DfuseTestBase
+from env_modules import load_mpi
+from general_utils import DaosTestError
```

Comment:
Pylint is requesting this change (plus the removal of line 13, but I could not select both lines for some reason).  By ignoring it the error will just be passed on to the next person who modifies the file.
```suggestion
from general_utils import DaosTestError, run_command
```

Reply:
Fixed